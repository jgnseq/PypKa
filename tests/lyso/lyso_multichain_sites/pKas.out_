
   A      1    NTR    Not In Range
   B      1    ASP    Not In Range
   C     35    GLU    Not In Range
   D     35    ASP    Not In Range
   D     66    ASP    Not In Range
   D      1    CTR    Not In Range