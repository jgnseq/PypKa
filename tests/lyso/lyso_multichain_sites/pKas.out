    1 NTR 7.242283991665559    A  
    1 ASP 3.1153294367693944   B  
   35 GLU 4.622119696447051    C  
   35 ASP 2.3149249683601516   D  
   66 ASP 1.8855327203893997   D  
    1 CTR 2.395745255014842    D  
